src/views/Sjofn/Practice43.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div>
+    Practice 4-3
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Practice3',
+  data () {
+    return {
+      name: '',
+      min: '',
+      max: '',
+      a: ''
```

Comment:
Do not create `data` values for local variables.

P.S.: 'local variable' means a value that is not visible outside of block of code, like an argument of a function or something declared **within** function. 
Please, read and learn the following:
- Javascript variables
- Javascript functions and parameters
- Javascript "pure function"
https://medium.com/@jamesjefferyuk/javascript-what-are-pure-functions-4d4d5392d49c

Reply:
Remove redundant `data` values.